.gitignore

```diff
@@ -119,3 +119,6 @@ compile_commands.json
 # rpath_check
 tt_eager/tt_lib/.rpath_checked*
 ttnn/ttnn/.rpath_checked
+
+# exclude boost from CPM
+tt_metal/third_party/boost
```

Comment:
I'm thinking we create a `.cpmcache` folder in root rather than pollute this folder

Reply:
Ok sounds good.